### PR TITLE
Crossing off some items in the TODO list

### PR DESCRIPTION
**Pull request by vchong**
@ruchi393 please review

### CODE REVIEW

File: keymaster/ta/attestation.c [hunk 1]
```diff
@@ -1059,39 +1059,41 @@ TEE_Result TA_generate_UniqueID(uint64_t T, uint8_t *appID, uint32_t appIDlen,
...
 #ifdef CFG_ATTESTATION_PROVISIONING
-keymaster_error_t TA_SetAttestationKey(TEE_Param params[TEE_NUM_PARAMS])
+TEE_Result TA_SetAttestationKey(TEE_Param params[TEE_NUM_PARAMS])
 {
 	uint8_t *in = NULL;
 	uint8_t *in_end = NULL;
 	size_t  in_size = 0;
 	keymaster_blob_t input = EMPTY_BLOB;	/* IN */
-	keymaster_algorithm_t algorithm;			/* IN */
+	keymaster_algorithm_t algorithm;	/* IN */
```
Comment (ruchi393) [inline keymaster/ta/attestation.c]: Can you initialize it also ?

File: keymaster/ta/attestation.c [hunk 2]
```diff
@@ -1100,69 +1102,74 @@ keymaster_error_t TA_SetAttestationKey(TEE_Param params[TEE_NUM_PARAMS])
...
 }
 
-keymaster_error_t TA_AppendAttestationCertKey(TEE_Param params[TEE_NUM_PARAMS])
+TEE_Result TA_AppendAttestationCertKey(TEE_Param params[TEE_NUM_PARAMS])
 {
 	uint8_t *in = NULL;
 	uint8_t *in_end = NULL;
 	size_t  in_size = 0;
 	keymaster_blob_t input = EMPTY_BLOB;	/* IN */
 	uint32_t algorithm;			/* IN */
```
Comment (ruchi393) [inline keymaster/ta/attestation.c]: Can you initialize this variable.

**Comment by ruchi393** [comment]
typo in commit - 
km: test: add inititial check for SetUp() 
%s/inititial/initial

Few minor unrelated comments. Not sure about the removal of TODO comments but you know better on that. Rest looks fine to me

**Comment by vchong** [comment]
@ruchi393 All comments addressed and also added a bit more changes. Please review again. Thanks!

**Comment by vchong** [comment]
@ruchi393 ping?

**Comment by ruchi393** [comment]
LGTM

`Acked-by: Ruchika Gupta <ruchika.gupta@linaro.org>`

**Pull request merged**